### PR TITLE
Fix servers ambience sound

### LINKED ISSUE #12441: R&D server's ambience doesn't stop if it's unpowered
## Description

Agony
**Reproduction**

Unwrenching or depower it
Still makes that awful noise
**Screenshots**

**Additional context**

### LINKED ISSUE #21483: Some devices seem to still make sounds despite a lack of power.
## Description
Certain devices that have ambient sounds still make sounds even when they are unpowered.

**Reproduction**
-Find an R&D server.
-Unanchor it so it has no power.
-Listen to the fans as they whirr harder than airboat with overweight tourists.

**Screenshots**
image
ITS WHIRRING

**Additional context**
Not really important but I found this when I was trying to test if punching mice while also trying to hold/drag them crashes the game. I don't know how to set up solars. No, I won't learn.

### PR DESCRIPTION
**Pull request by Winkarst-cpu**
## About the PR

Added ambient sound to camera routers and telecommunication servers. Added component to turn ambient sound off once the server/router is unpowered. Made the wireless camera router base abstract.

## Why / Balance

The wireless camera router base was probably intended to be abstract. Fixed #12441. Fixed #21483.

## Technical details

- Added ``AmbientSoundComponent`` to the telecommunication server and camera router.

- Added ``AmbientOnPoweredComponent`` to the servers and routers.

- Made the wireless camera router base to be abstract via ``abstract: true``.

## Media

https://github.com/user-attachments/assets/87f173f1-490a-4e5f-af1c-a444a083c3fb

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added ambience to the camera routers and telecommunication servers.
- fix: Now server's and router's ambience stops once they are unpowered.

### CODE REVIEW

**Comment by lzk228** [comment]
probably should be better implemented and not "hardcoded" to servers, and give it to all entities that should ambient when powered

**Comment by Winkarst-cpu** [comment]
> probably should be better implemented and not "hardcoded" to servers, and give it to all entities that should ambient when powered

I'm such a retard, I'm completly forgot about ``AmbientOnPoweredComponent``.

File: Content.Server/Medical/CrewMonitoring/CrewMonitoringServerSystem.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using Content.Server.DeviceNetwork;
+using Content.Server.DeviceNetwork;
```
Comment (deathride58) [inline Content.Server/Medical/CrewMonitoring/CrewMonitoringServerSystem.cs:1]: ?

Comment (metalgearsloth) [reply Content.Server/Medical/CrewMonitoring/CrewMonitoringServerSystem.cs:1]: Looks like casualty from before cleanup.

**Pull request merged**